packages/calcite-components/src/utils/variableValue.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ *
+ * Sets the value of a CSS variable to a test value.
+ * This is useful for testing themed components.
+ *
+ * @param token - the token as a CSS variable
+ * @returns string - the new value for the token
+ */
+export function setVariableValue(token: string): string {
+  const tokenValueMap = {
+    background$: "rgb(252, 244, 52)",
+    "text-color$": "rgb(239,118,39)",
+    "border-color$": "rgb(156, 89, 209)",
+    "background-color$": "rgb(44, 44, 44)",
+    color$: "rgb(0, 191, 255)",
+    highlighted$: "rgb(255, 105, 180)",
+    selected$: "rgb(255, 255, 255)",
+    shadow$:
+      "rgb(255, 255, 255) 0px 0px 0px 4px, rgb(255, 105, 180) 0px 0px 0px 5px inset, rgb(0, 191, 255) 0px 0px 0px 9px",
+    "z-index$": "42",
+    "(size|space)$": "42px",
```

Comment:
@alisonailea @jcfranco I noticed this one isn't working for a variable with `space-x` or `space-y`. Can we get it fixed?

Reply:
We'll also need to add `offset`, `offset-x`, `offset-y`

Reply:
feel free to add test examples where you need them too